Foundation/ProcessInfo.swift

```diff
@@ -192,3 +192,10 @@ open class ProcessInfo: NSObject {
         return NSFullUserName()
     }
 }
+
+// SPI for TestFoundation
+public extension ProcessInfo {
```

Comment:
Just out of curiosity, why `public` and not `internal`? If it's for unit tests, with `@testable import` you can still use internal APIs.

Reply:
Because that allows us to use this without a testable import.

Reply:
Nope. Please use `internal`. (We went over this.)

Reply:
(The tests will have to use `@testable import`.)